tools/sync.js

```diff
@@ -330,6 +458,11 @@ if (config.patch === true){
   runPatcher(config);
 }
 
+// check NORICHLIST env
+if (process.env.NORICHLIST) {
+  config.useRichList = false;
```

Comment:
This won't work since your code above only shows Accounts if `config.settings.useRichList` is toggled true in the HTML. I guess you'll need to pick if you want `useRichList` in `config` parent JSON bit or inside `settings`.